### DISCUSSION TITLE
Rolename assignment changed

### DISCUSSION THREAD

**Discussion by wbrussell** [ServiceStack/Discuss#69] [Q&A]
Updated from 5.11 to 6.10 and this broke my admin role name assignment.  Is there a new way to do this?

RoleNames.Admin = SecurityRoles.AdminRole;  //change default admin role to match my role name "administrator"

https://github.com/ServiceStack/ServiceStack/commit/298b3b1410b267e69655f0417c7e5b7726a90eb8

**Comment by mythz** [comment]
No it needs to be a constant in order to be usable from static contexts like in attributes.

You can give another role close to Admin's super user access by overriding the `HasAllRolesAsync` implementation in your custom User Session to include your custom role check as well:

https://github.com/ServiceStack/ServiceStack/blob/ef21a950fd8c2c9836d18039743e1abe5f631fef/ServiceStack/src/ServiceStack/AuthUserSession.cs#L121
